addons/website_slides/static/src/js/slides_course_quiz.js

```diff
@@ -6,6 +6,7 @@ odoo.define('website_slides.quiz', function (require) {
     var session = require('web.session');
 
     var CourseJoinWidget = require('website_slides.course.join.widget').courseJoinWidget;
+    var FinishQuizModal = require('website_slides.quiz.finish');
```

Comment:
Please name it SlideQuizFinishModal like in the widget definition to ease greping.